terraform/deployment.tf

```diff
@@ -279,6 +279,30 @@ EOF
 
 }
 
+resource "aws_ecr_lifecycle_policy" "ecr_expire" {
+  repository = aws_ecr_repository.discourse.name
+
+  policy = <<EOF
+{
+  "rules": [
+    {
+      "rulePriority": 1,
+      "description": "Expire untagged image older than ${var.ecr_expire_days} days",
+      "selection": {
+        "tagStatus": "untagged",
```

Comment:
can we change "untagged" for "any"?

Reply:
Resolved in 49cf9d794c22876763850b0538f47fb3dca49f09